cmd/upgrade.go

```diff
@@ -68,3 +66,14 @@ func upgrade(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func upgradePackage(cmd *cobra.Command, args []string) error {
+
+	err := core.NixUpgradePackage(args[0])
+	if err != nil {
+		return err
+	}
+	cmdr.Success.Println(apx.Trans("nixupgrade.success"))
+	return nil
+
+}
```

Comment:
Isn't this part of the other PR? I think you added the changes from #122 to this one by accident.

Reply:
yes, but if we merge the other first my senior citizen moment won't matter.